XCUITests/DownloadFilesTests.swift

```diff
@@ -63,6 +65,18 @@ class DownloadFilesTests: BaseTestCase {
         XCTAssertTrue(app.tables.cells.staticTexts[testFileSize].exists)
     }
 
+    func testDownloadBLOBFile() {
+        downloadBLOBFile()
+        navigator.goto(BrowserTabMenu)
```

Comment:
We may need to add a waitForNoExistence for the snackBar menu with the download file info before going to the BrowserTabMenu just in case the Download has not finished or the test fails to go here if that menu is still on the screen

Reply:
Added 👍